### ISSUE TITLE
Chat area too small on Android

### ISSUE THREAD

**Issue by TheRealCryptKeeper** [byteball/obyte-gui-wallet#444]
![Screenshot_Android_2-7-0](https://user-images.githubusercontent.com/25538715/56494140-97e87280-64f1-11e9-8963-5babb17e8ceb.png)

The chat area height is off by a few pixels on Android, making some controls (like 'Remove a device' in Contacts) hard to use.

Screenshot attached.

 - Device: Google Pixel 2 
 - OS: Android 9
 - Display Resolution 1920 x 1080
 - Obyte version 2.7.0

**Comment by tarmo888** [comment]
I think you mean that there is not enough padding on mobile when scrolled to the bottom of Contacts page.
On Android 8.1, the "Remove a device" is also quite close to the navigation, but not as close as yours. Maybe because I have physical buttons for the OS features (Back, Home, Recent Apps).

**Comment by TheRealCryptKeeper** [comment]
Exactly. Also, the entry field in bot's chat is not as high as in the windows wallet but this doesn't hinder usability as much as with 'Remove a device'. Most of the time I've tried to tap on remove I ended up in the receive dialog.
